manifests/v/vim/vim/8.2.3197/vim.vim.installer.yaml

```
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: vim.vim
+PackageVersion: 8.2.3197
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
+  InstallerUrl: https://github.com/vim/vim-win32-installer/releases/download/v8.2.3197/gvim_8.2.3197_x64.exe
+  InstallerSha256: B6930474E6C89D87D6575BB2D5BE9CAE1AA2C7201082A78D10C9D0886F46BBE8
```

Signed version just came out 👍 

```suggestion
  InstallerUrl: https://github.com/vim/vim-win32-installer/releases/download/v8.2.3197/gvim_8.2.3197_x64_signed.exe
  InstallerSha256: 0990F8CC5604C691ABFCA691F13C48B2FE711EAADE6D876A09418FCB8FCFB9E0
```